### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by kkseven07** [rtkhanas/react-qr-code#45]
./~/react-qr-code/lib/components/QRCodeSurfaceCell/index.js
Module not found: Error: Cannot resolve module 'react-art-svg-renderer'

**Comment by ships** [comment]
This may be related, that when I try to install with Yarn, I get this message:

```
warning " > react-qr-code@0.1.2" has unmet peer dependency "react@^16.0.0".
warning " > react-qr-code@0.1.2" has unmet peer dependency "react-art-svg-renderer@^0.0.1-rc.8".
warning " > react-qr-code@0.1.2" has unmet peer dependency "react-native@^0.49.3".
```

and one sees when trying to install the art-svg-renderer:
```
warning " > react-qr-code@0.1.2" has unmet peer dependency "react@^16.0.0".
warning " > react-qr-code@0.1.2" has unmet peer dependency "react-native@^0.49.3".
warning " > react-art-svg-renderer@0.0.1-rc.8" has unmet peer dependency "art@^0.10.1".
warning " > react-art-svg-renderer@0.0.1-rc.8" has unmet peer dependency "react@^15.4.2".
```

I am new to Yarn but I interpret this to mean, the `react-art-svg-renderer` requires a conflicting version of `react` to the one required by this package.

**Comment by rtkhanas** [comment]
✅ I just added a `react-art-svg-renderer` package to a list of dependencies and published `v0.1.3` to NPM. Please check.
